<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>665</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>665</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>665</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lb_Logo">
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/Image/Join.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="Btn_FindPW">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>320</y>
         <width>125</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Image/Find_PW.png</normaloff>:/Image/Find_PW.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lb_Pw">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>100</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PW</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_ID">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>40</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PW_Edit">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>100</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>  비밀번호 입력</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_Join">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>260</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Image/Join_.PNG</normaloff>:/Image/Join_.PNG</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>260</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID_Edit">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>40</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>  아이디 입력</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_FindID">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>320</y>
         <width>125</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Image/Find_ID.png</normaloff>:/Image/Find_ID.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_Login">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>200</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Image/Login.PNG</normaloff>:/Image/Login.PNG</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>260</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLineEdit" name="PW_JoinEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>  비밀번호 입력</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_Join_NAME">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>이름</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_Join_PHONE">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>전화번호</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NAME_JoinEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>  이름 입력</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_confirmNICKNAME">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>270</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>중복확인</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_Join_NICKNAME">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>닉네임</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID_JoinEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>  아이디 입력</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_Join_ID">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_confirmJoin">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>380</y>
         <width>260</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Image/Join_.PNG</normaloff>:/Image/Join_.PNG</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>260</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="PHONE_JoinEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>  전화번호 입력</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NICKNAME_JoinEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>270</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>  닉네임 입력</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_Join_PW">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>70</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PW</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_JoinBack">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Image/Back.png</normaloff>:/Image/Back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_confirmID">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>중복확인</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_confirmPHONE">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>210</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>중복확인</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_ckID">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(115, 186, 184);</string>
       </property>
       <property name="text">
        <string>아이디를 입력하세요.</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_ckPW">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(115, 186, 184);</string>
       </property>
       <property name="text">
        <string>비밀번호를 입력하세요.</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>stackedWidget</zorder>
  <zorder>lb_Logo</zorder>
 </widget>
 <tabstops>
  <tabstop>ID_Edit</tabstop>
  <tabstop>PW_Edit</tabstop>
  <tabstop>Btn_Login</tabstop>
  <tabstop>Btn_Join</tabstop>
  <tabstop>Btn_FindID</tabstop>
  <tabstop>Btn_FindPW</tabstop>
  <tabstop>ID_JoinEdit</tabstop>
  <tabstop>Btn_confirmID</tabstop>
  <tabstop>PW_JoinEdit</tabstop>
  <tabstop>NAME_JoinEdit</tabstop>
  <tabstop>PHONE_JoinEdit</tabstop>
  <tabstop>Btn_confirmPHONE</tabstop>
  <tabstop>NICKNAME_JoinEdit</tabstop>
  <tabstop>Btn_confirmNICKNAME</tabstop>
  <tabstop>Btn_confirmJoin</tabstop>
  <tabstop>Btn_JoinBack</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ID_Edit</sender>
   <signal>returnPressed()</signal>
   <receiver>PW_Edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PW_Edit</sender>
   <signal>returnPressed()</signal>
   <receiver>Btn_Login</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Login</sender>
   <signal>clicked()</signal>
   <receiver>LoginPage</receiver>
   <slot>LoginProgress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_JoinBack</sender>
   <signal>clicked()</signal>
   <receiver>LoginPage</receiver>
   <slot>gotoLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>-73</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Join</sender>
   <signal>clicked()</signal>
   <receiver>LoginPage</receiver>
   <slot>gotoJoin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ID_JoinEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginPage</receiver>
   <slot>CheckID(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_confirmID</sender>
   <signal>clicked()</signal>
   <receiver>LoginPage</receiver>
   <slot>SendJoinID()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PW_JoinEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginPage</receiver>
   <slot>CheckPW(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_confirmPHONE</sender>
   <signal>clicked()</signal>
   <receiver>LoginPage</receiver>
   <slot>CheckPHONE()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LoginProgress()</slot>
  <slot>gotoLogin()</slot>
  <slot>gotoJoin()</slot>
  <slot>CheckID(QString)</slot>
  <slot>SendJoinID()</slot>
  <slot>CheckPW(QString)</slot>
  <slot>CheckPHONE()</slot>
 </slots>
</ui>
